less/base.less

```diff
@@ -109,7 +109,7 @@ h3 {
 }
 
 .hidden {
-	display: none;
+	display: none !important;
```

Comment:
Why is this important needed? Best avoid those, if possible.

Reply:
To override ``#catalog article { display: inline-block; }``
Rules bound to an id override class rules so the !important is needed.